### PR TITLE
Update Portfolio Optimizer [build:portfolio_optimizer]

### PR DESCRIPTION
**Pull request by philippjfr**
Screen Shot 2021-01-02 at 11 28 19 PM

### CODE REVIEW

File: portfolio_optimizer/anaconda-project.yml [hunk 1]
```diff
@@ -8,21 +8,19 @@ labels:
 - hvplot
 - holoviews
 
-user_fields: [labels, skip, maintainers]
+user_fields: [labels, skip, maintainers, user_fields, user_fields, user_fields, user_fields]
```
Comment (jbednar) [inline portfolio_optimizer/anaconda-project.yml:11]: This is from an anaconda-project bug, which I thought had been fixed, but maybe you have an older version? If it's the current anaconda-project version please file a bug report.

```suggestion
user_fields: [labels, skip, maintainers]
```

File: portfolio_optimizer/portfolio.py [hunk 2]
```diff
@@ -275,4 +281,11 @@ def plot_return_curve(table, investment, dates):
...
 
-pn.Row(pn.Column(widgets, explanation), pn.layout.Spacer(width=20), get_portfolio_analysis).servable()
+template = pn.template.MaterialTemplate(title="Portfolio Optimizer")
+
+template.sidebar.append(widgets)
+template.sidebar.append(pn.panel(explanation, sizing_mode='stretch_width'))
+
+template.main.append(pn.panel(get_portfolio_analysis))
+
+template.servable()
```
Comment (jbednar) [inline portfolio_optimizer/portfolio.py:291]:
```suggestion
template.servable()

```

**Comment by philippjfr** [comment]
Won't merge until all the updated dependencies are available. As an aside I think it'd be nice to add a GH Action to update the lock files.

**Comment by philippjfr** [comment]
The NumPy linux-64 build that is locked here appears to be very slow, switching to conda-forge reduces load times by 5-10x. Will need further investigation and potentially have to be reported to the conda team.

**Comment by philippjfr** [comment]
Pinned to latest and relocked.

**Pull request merged**